delete-user-data/functions/src/helpers.ts

```diff
@@ -25,3 +28,26 @@ export const getDatabaseUrl = (
 
   return `https://${selectedDatabaseInstance}.${selectedDatabaseLocation}.firebasedatabase.app`;
 };
+
+export const hasValidUserPath = async (
+  ref: DocumentReference,
+  path: string,
+  uid: string
+): Promise<boolean> => {
+  /** Check path for valid user id */
+  if (path.includes(uid)) return Promise.resolve(true);
```

Comment:
I think you can just `return true` here, the fact that it's an async function should pick up the promise

Reply:
similarly on line 46 and 52